CMakeLists.txt

```diff
@@ -13,6 +13,8 @@ option(USE_QT5 "Use Qt5 even if Qt6 found" OFF)
 option(VCPKG "Use vcpkg for release mode dependencies" OFF)
 option(VCPKG_BUILDTREES_ROOT "Use custom vcpkg buildtrees" "")
 
+option(CXX_QT_BUILD_TESTS "Build the tests for CXX-Qt" ON)
```

Comment:
do we need the `CXX_QT` prefix ? Could it just be `BUILD_TESTS` as it's kinda common to have `BUILD_TESTS` and `BUILD_EXAMPLES` i thought?

Reply:
i felt it was better to be more descriptive, but maybe it doesnt matter

Reply:
As we currently have `USE_QT5` and `VCPKG` we probably don't need the prefix, as either they should all have it or they should all not it to be consistent

Reply:
If we go with a generic name, make sure it's `BUILD_TESTING` as seems to be more of the standard across CMake projects

Reply:
Aha thanks, i knew i had seen something like that before :sweat_smile: